packages/api/lib/cwSfExecutionEventUtils.js

```diff
@@ -23,6 +19,15 @@ const executionStatusToWorkflowStatus = (executionStatus) => {
   return statusMap[executionStatus];
 };
 
+const getTaskExitedEventOutput = (event) =>
```

Comment:
Moving these here seems right, but there's still a unit test here:

https://github.com/nasa/cumulus/blob/CUMULUS-1684-remove-S3KeyPairProvider-2/packages/common/tests/test-execution-history.js


Why is this passing in CI?     Caching?

Reply:
Anyway, I'd expect [`6f5a252` (#1800)](https://github.com/nasa/cumulus/pull/1800/commits/6f5a2525d2f10fbd4c29d47ede3e8cd07dea521e) should fix it.

Reply:
Yeah, that was my mistake. Fixed now.